<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>node1</class>
 <widget class="QWidget" name="node1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 85, 255);
border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="Header">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>5</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>node #</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PhotoLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>160</width>
     <height>213</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="GroupLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Group1CB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Group2CB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ModeLbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ModeCB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Blue</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="RoleButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Role</string>
   </property>
  </widget>
  <widget class="QLabel" name="DeletedLbl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ActivateButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Activated</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SimulateButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
